backend/server/principal.go

```diff
@@ -66,6 +68,17 @@ func (s *Server) registerPrincipalRoutes(g *echo.Group) {
 			principal.ServiceKey = password
 		}
 
+		if s.MetricReporter != nil {
+			s.MetricReporter.Report(&metric.Metric{
+				Name:  metricAPI.MemberCreateMetricName,
+				Value: 1,
```

Comment:
Can we use the value to record this is the nth user?

Reply:
We can use the label to store this info, updated.